.github/workflows/build.yml

```diff
@@ -107,36 +112,38 @@ jobs:
         uses: microsoft/setup-msbuild@v1.1
 
       - name: Create Build Environment
-        run: cmake -E make_directory ${{ runner.workspace }}/build
+        run: cmake -E make_directory ${{ github.workspace }}/build
 
       - name: Configure CMake
         shell: bash
-        working-directory: ${{ runner.workspace }}/build
+        working-directory: ${{ github.workspace }}/build
         run: |
+          OPTIONS="-DCMAKE_BUILD_TYPE=${{ matrix.build_type }} -DCMAKE_INSTALL_PREFIX=bin"
           if [ "${{ matrix.portable }}" == "Portable" ]; then
-            OPTIONS="-DCMAKE_BUILD_TYPE=${{ runner.build_type }} -DBuildPortableVersion=ON -DCMAKE_INSTALL_PREFIX=bin"
+            OPTIONS+=" -DBuildPortableVersion=ON"
           else
-            OPTIONS="-DCMAKE_BUILD_TYPE=${{ runner.build_type }} -DBuildPortableVersion=OFF -DCMAKE_INSTALL_PREFIX=bin"
+            OPTIONS+=" -DBuildPortableVersion=OFF"
           fi
+          OPTIONS+=" -DBuildJK2SPEngine=ON -DBuildJK2SPGame=ON -DBuildJK2SPRdVanilla=ON"
           cmake $GITHUB_WORKSPACE -T v141_xp -A ${{ matrix.platform }} $OPTIONS
 
       - name: Build
-        working-directory: ${{ runner.workspace }}/build
+        working-directory: ${{ github.workspace }}/build
         shell: bash
         run: cmake --build . --config ${{ matrix.build_type }}
 
       - name: Install
-        if: ${{ matrix.build_type == 'Release' }} 
-        working-directory: ${{ runner.workspace }}/build
+        if: ${{ matrix.build_type == 'Release' }}
+        working-directory: ${{ github.workspace }}/build
         shell: bash
         run: |
           cmake --install . --config ${{ matrix.build_type }}
-          
+
       - uses: actions/upload-artifact@v3
-        if: ${{ matrix.build_type == 'Release' }} 
+        if: ${{ matrix.build_type == 'Release' }}
         with:
-          name: OpenJK-windowsxp-${{ matrix.arch }}-${{ matrix.build_type }}-${{ matrix.portable }}
-          path: ${{ runner.workspace }}/build/bin
+          name: windowsxp-${{ matrix.arch }}-${{ matrix.build_type }}-${{ matrix.portable }}
+          path: ${{ github.workspace }}/build/bin
```

Comment:
FIXME: reintroduce game prefix, maybe split upload step into two? why is this different to the other runners?